cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+import { faker } from "@faker-js/faker";
+const name = faker.name.fullName();
+const company = faker.company.bs();
+const address = faker.address.buildingNumber();
+const phone = faker.phone.number("984######");
+const email = faker.internet.email();
+const city = faker.address.city();
```

Comment:
Insert in test spec if it is used in only one test spec. No need for global declaration